src/System.Private.CoreLib/src/System/Activator.cs

```diff
@@ -147,6 +147,18 @@ public static object CreateInstance(Type type, BindingFlags bindingAttr, Binder
         }
 
         private const BindingFlags ConstructorDefault = BindingFlags.Instance | BindingFlags.Public | BindingFlags.CreateInstance;
+
+        public static System.Runtime.Remoting.ObjectHandle CreateInstance(string assemblyName, string typeName) { throw new PlatformNotSupportedException(); }
+
+        public static System.Runtime.Remoting.ObjectHandle CreateInstance(string assemblyName, string typeName, bool ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, object[] args, System.Globalization.CultureInfo culture, object[] activationAttributes) { throw new PlatformNotSupportedException(); }
+
+        public static System.Runtime.Remoting.ObjectHandle CreateInstance(string assemblyName, string typeName, object[] activationAttributes) { throw new PlatformNotSupportedException(); }
+
+        public static System.Runtime.Remoting.ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName) { throw new PlatformNotSupportedException(); }
+
```

Comment:
This is 300+ characters long line. Could you please add `using System.Runtime.Remoting` at the top of the file, and re-format these to be on multiple shorter lines